docs/source/how-to/adaptive.rst

```diff
@@ -11,13 +11,24 @@ two situations:
 1.  The user may not be using the cluster, or perhaps they are busy
     interpreting a recent result or plot, and so the workers sit idly,
     taking up valuable shared resources from other potential users
-2.  The user may be very active, and is limited by their original allocation.
+2.  The user may be very active, and is limited by their original allocation,
+    particularly in the case of bursty workloads
+
+.. image:: ../images/bursty-workloads.svg
```

Comment:
If you are using this figure to represent just the notion of a bursty workload, then I think it would make more sense not to have the adaptive part in this figure.

Reply:
also the sideways words on both of these are very small. Maybe they should get  larger width.

Reply:
good point. I removed the bursty workloads image from this page because I think the page really focuses on the adaptive part. Do you know of another place where the bursty workloads image might be a good fit in the docs?

Reply:
I can't think of where it would go right now, but we can still include it in the images.

Reply:
ok! Removed here, but added to dask/marketing